README.md

```diff
@@ -0,0 +1,90 @@
+# Purpose of this Repository:
+This repository is intended as a template to standardize and ease the initialization process for Technical Reports from members of the Advanced Reactors and Fuel Cycles group at the University of Illinois at Urbana-Champaign. While this version of the template is geared towards an ARFC Member's report, it can be adapted to any individual's needs.
+
+# How to Use this Template:
+
+This repository serves as a template, for information on making and using a repository go to https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template.
+
+### The overview of this process is:

```

Comment:
I don't think you need the extra heading since the last one is already on how to use the template

Reply:
I agree, I've now re-ordered that section.